pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -1373,6 +1391,17 @@ class HdRprApiImpl {
 
     void RenderImpl(HdRprRenderThread* renderThread, std::vector<std::pair<void*, size_t>> const& outputRenderBuffers) {
         bool stopRequested = false;
+
+		// Made temporarily just to have working interop
+		if (m_rprContextMetadata.pluginType == rpr::kPluginHybrid) {
+			std::cout << "[Plugin] Render\n";
+			auto status = m_rprContext->Render();
+			rprContextFlushFrameBuffers_func rprContextFlushFrameBuffers;
+			rprContextGetFunctionPtr(m_rprContext->Handle(), RPR_CONTEXT_FLUSH_FRAMEBUFFERS_FUNC_NAME, (void**)(&rprContextFlushFrameBuffers));
+			rprContextFlushFrameBuffers(m_rprContext->Handle());
+			return;
+		}
```

Comment:
This should be called after each `Render` or just once? What's the logic behind this flushing?

Reply:
Still figuring out. Render should be in loop till enough render passes reached. Then flush function should be called. Flush  make semaphores signalled and tells us we can render next frame.